Sources/Sentry/SentryAppState.m

```diff
@@ -22,7 +22,7 @@ - (instancetype)initWithReleaseName:(NSString *)releaseName
         NSTimeInterval interval = round(systemBootTimestamp.timeIntervalSince1970);
         _systemBootTimestamp = [[NSDate alloc] initWithTimeIntervalSince1970:interval];
 
-        _isActive = NO;
+        _isActive = YES;
```

Comment:
The default of `NO` seemed kinda weird.. whenever we create a new app state, the app is running, right?